Makefile

```diff
@@ -73,11 +75,16 @@ clean: tools
 	-rm -rf coverage.txt
 
 .PHONY: build
-build: clean deps linux darwin windows
+build: clean deps
+	rm -rf bin/*
```

Comment:
This line should be in `clean` goal?

Reply:
In the local test ， I build some different version docker images and push them to a private repository. Because of using the find command in the start script， the docker started failed when having history file. If needed to the original version, I would rollback it.

Reply:
I polish the docker image, please review https://github.com/apache/skywalking-satellite/pull/28 and rebase this on that after merging it